kbase-extension/static/kbase/css/kbaseNarrative.css

```diff
@@ -2481,7 +2481,7 @@ button.kb-data-obj {
     moz-box-shadow: 1px 1px 3px #aaa;
 }
 .kb-data-list-add-data-text-button:hover {
-    background-color: #DC3C31;
+    background-color: #4379B1;
```

Comment:
Same here - if we don't want to tweak the color on hover, this whole block could be removed.